BranchMetrics/ios-branch-deep-linking-attribution
# SDK-1982 SKAN Simplification DB Updates

## Reference
SDK-1982 -- [iOS] SKAN Simplification DB updates .
https://branch.atlassian.net/browse/SDK-1982 

## Summary
* Added parameter for current SKAN Window (skan_postback_index) in /v1/open and /v2/event requests
* Updated function 'shouldCallPostbackForDataResponse' to handle negative conversion values for second and third SKAN Windows.

## Motivation
Support SKAN 4.0 DB Simplication.

## Type Of Change


## Testing Instructions
With Every INSTALL/OPEN/EVENT requests, Verify using logs SDK is sending current SKAN window number. 
Verify in response , SDK is getting back conversion values.
Some of the samples are here -
https://www.notion.so/brancheng/SDK-SKAN-Testing-Results-5a74041cd815457292725673097f66c4










cc @BranchMetrics/saas-sdk-devs for visibility.
